db/migrate/20230621153809_add_admin_to_users.rb

```diff
@@ -0,0 +1,6 @@
+
+class AddAdminToUsers < ActiveRecord::Migration[7.0]
+    def change
+      add_column :users, :admin, :boolean
+    end
```

Comment:
- [ ] Not required to add `admin` column , for `admin`  while creating user ,just pass `role = admin`. Kindly remove this.